jsk_fetch_robot/jsk_fetch_startup/launch/rosbag_play.launch

```diff
@@ -0,0 +1,95 @@
+
+  <!-- decompress -->
+  <node name="rgb_decompress" pkg="image_transport" type="republish"
+        args="compressed in:=$(arg RGB_IMAGE) out:=$(arg RGB_IMAGE)" />
+  <node name="depth_decompress" pkg="image_transport" type="republish"
+        args="compressedDepth in:=$(arg DEPTH_IMAGE) out:=$(arg DEPTH_IMAGE)" />
+  <node pkg="nodelet" type="nodelet" name="point_cloud_xyzrgb"
+        args="load depth_image_proc/point_cloud_xyzrgb $(arg manager)" output="screen" >
+    <remap from="rgb/camera_info" to="$(arg RGB_CAMERA_INFO)" />
+    <remap from="rgb/image_rect_color" to="$(arg RGB_IMAGE)" />
+    <remap from="depth_registered/image_rect" to="$(arg DEPTH_IMAGE)" />
+    <remap from="depth_registered/points" to="/head_camera/depth_registered/throttled/points" />
+    <rosparam>
+      queue_size: 100
+    </rosparam>
+  </node>
+  <node name="resize_rgb" pkg="nodelet" type="nodelet"
+        args="load image_proc/resize $(arg manager)" output="screen">
+    <remap from="image" to="$(arg RGB_IMAGE)" />
+    <remap from="camera_info" to="$(arg RGB_CAMERA_INFO)" />
+    <remap from="~image" to="$(arg QUAT_RGB_IMAGE)" />
+    <remap from="~camera_info" to="$(arg QUAT_RGB_CAMERA_INFO)" />
+    <rosparam>
+      scale_width: 0.25
+      scale_height: 0.25
+    </rosparam>
+  </node>
+  <node name="resize_cloud" pkg="nodelet" type="nodelet"
+        args="load jsk_pcl/ResizePointsPublisher $(arg manager)" output="screen">
+    <remap from="~input" to="/head_camera/depth_registered/throttled/points" />
+    <remap from="~output" to="/head_camera/depth_registered/quater/throttled/points" />
+    <rosparam>
+      step_x: 4
+      step_y: 4
+    </rosparam>
+  </node>
+
+  <!-- relay -->
+  <node name="rgb_image_relay" pkg="topic_tools" type="relay"
+        args="$(arg RGB_IMAGE) /head_camera/rgb/image_rect_color" />
+  <node name="rgb_info_relay" pkg="topic_tools" type="relay"
+        args="$(arg RGB_CAMERA_INFO) /head_camera/rgb/camera_info" />
+  <node name="quat_rgb_image_relay" pkg="topic_tools" type="relay"
+        args="$(arg QUAT_RGB_IMAGE) /head_camera/rgb/quater/image_rect_color" />
+  <node name="quat_rgb_info_relay" pkg="topic_tools" type="relay"
+        args="$(arg QUAT_RGB_CAMERA_INFO) /head_camera/rgb/quater/camera_info" />
+  <node name="depth_image_relay" pkg="topic_tools" type="relay"
+        args="$(arg DEPTH_IMAGE) /head_camera/depth_registered/image_rect" />
+  <node name="depth_info_relay" pkg="topic_tools" type="relay"
+        args="$(arg DEPTH_CAMERA_INFO) /head_camera/depth_registered/camera_info" />
+  <node name="points_relay" pkg="topic_tools" type="relay"
+        args="/head_camera/depth_registered/throttled/points /head_camera/depth_registered/points" />
+  <node name="quat_points_relay" pkg="topic_tools" type="relay"
+        args="/head_camera/depth_registered/quater/throttled/points /head_camera/depth_registered/quater/points" />
+
+  <!-- latched topic republisher -->
```

Comment:
@knorth55 
Why do you republish latched topics?
Without this script, I can rostopic echo latched topic.

For example,
```bash
# terminal 1
roscore
# terminal 2
rosbag play 20200619100612_go_to_kitchen_rosbag.bag
# terminal 3
rostopic echo /map --noarr
header:                                            
  seq: 0                                           
  stamp:                                           
    secs: 1592506865                               
    nsecs: 636232232                               
  frame_id: "map"                                  
info:                                              
  map_load_time:                                   
    secs: 1592506865                               
    nsecs: 636231939                               
  resolution: 0.0500000007451                      
  width: 1200                                      
  height: 1200                                     
  origin:                                          
    position:                                      
      x: -13.0                                     
      y: -48.5                                     
      z: 0.0                                       
    orientation:                                   
      x: 0.0                                       
      y: 0.0                                       
      z: 0.0                                       
      w: 1.0                                       
data: "<array type: int8, length: 1440000>"        
---     


```

Reply:
@sktometometo can you give us the situation which you need this node?